packages/legend-extension-dsl-serializer/src/components/studio/BindingElementEditor.tsx

```diff
@@ -0,0 +1,485 @@
+      value: binding.contentType,
+      label: binding.contentType,
+    };
+    return (
+      <div className="binding-general-editor">
+        <div className="binding-general-editor__section__header__label">
+          Content Type
+        </div>
+        <div>
+          <CustomSelectorInput
+            className="binding-general-editor__section__dropdown"
+            disabled={isReadOnly}
+            options={contentTypeOptions}
+            onChange={onContentTypeChange}
+            value={selectedContentType}
+            placeholder="Choose a content type"
+            darkMode={true}
+          />
+        </div>
+        <div className="binding-general-editor__section__header__label">
+          Schema Set
+        </div>
+        <div className="binding-general-editor__content">
+          <CustomSelectorInput
+            className="binding-general-editor__section__dropdown"
+            disabled={isReadOnly}
+            ref={projectSelectorRef}
+            options={schemaSetOptions}
+            onChange={onSchemaSetChange}
+            value={selectedSchemaSet}
+            isClearable={true}
+            placeholder="Choose a schema set"
+            darkMode={true}
+          />
+        </div>
+        <div className="binding-general-editor__section__header__label">
+          Schema ID
+        </div>
+        <div className="binding-general-editor__content">
+          <CustomSelectorInput
+            className="binding-general-editor__section__dropdown"
+            disabled={isReadOnly}
+            ref={projectSelectorRef}
+            options={schemaIdOptions}
+            onChange={onSchemaIdChange}
+            value={selectedSchemaId}
+            isClearable={true}
+            placeholder="Choose a schema ID"
+            darkMode={true}
+          />
+        </div>
+      </div>
+    );
+  },
+);
+
+export const BindingEditor = observer(() => {
+  const editorStore = useEditorStore();
+  const editorState = editorStore.getCurrentEditorState(BindingEditorState);
+  const binding = editorState.binding;
+  const modelUnit = binding.modelUnit;
+  const isReadOnly = editorState.isReadOnly;
+  const selectedTab = editorState.selectedTab;
+  const changeTab =
+    (tab: BINDING_TAB_TYPE): (() => void) =>
+    (): void =>
+      editorState.setSelectedTab(tab);
+  const graph = editorStore.graphManagerState.graph;
+  const allowedElements = [
+    ...graph.ownProfiles,
+    ...graph.ownEnumerations,
+    ...graph.ownClasses,
+    ...graph.ownAssociations,
+    ...graph.ownFunctions,
+  ];
+  const elements = allowedElements.filter(
+    (element) =>
+      !modelUnit.packageableElementIncludes
+        .map((element) => element.value)
+        .includes(element) &&
+      !modelUnit.packageableElementExcludes
+        .map((element) => element.value)
+        .includes(element),
+  );
+  const allowAddingElement = !isReadOnly && Boolean(elements.length);
+  const addElement = (label: BINDING_MODEL_UNIT): void => {
```

Comment:
See the comment above, let's create simple pair `removeExclusion`, `addExclusion`, `removeInclusion`, `addInclusion`, or something like that.